.github/workflows/docs.yml

```diff
@@ -70,14 +69,15 @@ jobs:
           echo "gchat_webhook=${{ secrets.GCHAT_WEBHOOK }}" >> $GITHUB_ENV
         shell: bash
 
-      - name: Run Staged dependencies 🎦
-        uses: insightsengineering/staged-dependencies-action@v1
-        env:
-          GITHUB_PAT: ${{ secrets.REPO_GITHUB_TOKEN }}
+      - name: Setup R dependencies 🎦
+        uses: insightsengineering/setup-r-dependencies@v1
         with:
-          path: "./package"
-          enable-check: false
-          direction: upstream
+          github-token: ${{ secrets.REPO_GITHUB_TOKEN }}
```

Comment:
Why not GITHUB_PAT env ? as in here https://github.com/insightsengineering/r.pkg.template/pull/262 ?

Reply:
not yet merged but you got the point. Will change after the merge

Reply:
Alrighty!